src/css/components/modal.scss

```diff
@@ -0,0 +1,20 @@
+
+@import "../override_vars.scss";
+@import "../core.scss";
+
+// faux modal for now, but maybe real later?
+
+.actions {
+  padding: 3rem 2rem;
+  background-color: #DEEEEF; // DEEF!
+  margin: -2rem;
+}
+
+strong.inline-block {
```

Comment:
Does this have to be qualified with `strong`? Is this something within the `.actions` div? if so, it might make sense to namespace it under that, such as `.actions-inline-block` or `.actions-whatever`.